woocommerce/woocommerce-android
# Issue/5459 simple payments cash

Closes #5459 by adding the ability to accept a cash payment and mark the order as completed. 

To test:

* Create a simple payment of any amount
* Tap Done
* Tap Take payment
* Confirm the dialog
* Observe that the dialog closes, you're returned to the order list, and the order has been marked completed

![Screenshot_20211229_155734](https://user-images.githubusercontent.com/3903757/147702941-927fc7ff-41c2-4e73-a1eb-f14ebbdb4eaf.png)
